lib/output/utils.js

```diff
@@ -29,10 +29,7 @@ function newObjectInRealm(globalObject, object) {
 const wrapperSymbol = Symbol("wrapper");
 const implSymbol = Symbol("impl");
 const sameObjectCaches = Symbol("SameObject caches");
-const ctorRegistrySymbol = Symbol.for("[webidl2js]  constructor registry");
-
-// This only contains the intrinsic names that are referenced from the `ctorRegistry`:
-const intrinsicConstructors = ["Array"];
+const ctorRegistrySymbol = Symbol("[webidl2js] constructor registry");
```

Comment:
Hmm, I see. Probably then the problem is that I was testing only upgrading webidl2js for jsdom, but not domexception and whatwg-url. I will have to test upgrading all three. Or maybe it would be best to add a major version number to the symbol description so that if we do mix versions it doesn't cause problems.